bokeh/sphinxext/bokehjs_content.py

```diff
@@ -57,15 +57,13 @@
 # External imports
 from docutils import nodes
 from docutils.parsers.rst.directives import unchanged
-from sphinx.directives.code import CodeBlock, container_wrapper, dedent_lines
+from sphinx.directives.code import CodeBlock
 from sphinx.errors import SphinxError
-from sphinx.locale import __
-from sphinx.util import logging, parselinenos
-from sphinx.util.nodes import set_source_info
+from sphinx.util import logging
 
 # Bokeh imports
 from .templates import BJS_CODEPEN_INIT, BJS_EPILOGUE, BJS_HTML, BJS_PROLOGUE
-from .util import get_sphinx_resources
+from .util import get_codeblock_node, get_sphinx_resources
 
 if False:
```

Comment:
As long as we are editing this file we could fix this hack. The proper way to do this is with
```
if TYPE_CHECKING:
```
and then I think flake8 might be smart enough that the `# noqa` are no longer needed too

Reply:
Sure thing. Also, I have flake8 installed locally so I can re-run it on the files. It wasn't giving me the errors that the codebase CI caught.